core/core-generic-ops.el

```diff
@@ -0,0 +1,144 @@
+(require 'evil)
```

Comment:
While Spacemacs is based on evil we don't require it in `core` we can put the `evil` dependent part in an `with-eval-after-load`.